enctests/testframework/utils/utils.py

```diff
@@ -17,14 +17,14 @@
 # Which vmaf model to use
 VMAF_HD_MODEL = os.getenv(
     'VMAF_MODEL',
-    f'{os.path.dirname(__file__)}/../../tools/vmaf-2.3.1/model/'
-) + "/vmaf_v0.6.1.json"
+    VMAF_LIB_DIR
+) + "/model/vmaf_v0.6.1.json"
```

Comment:
Should we rather just build a path here since providing the intended "VMAF_MODEL" variable won't cover both 4k and HD in any case?
It might spawn some updates other places we may be using "VMAF_MODEL" though.

``` python
VMAF_MODEL_VERSION = os.getenv('VMAF_MODEL_VERSION', 'v0.6.1')
VMAF_HD_MODEL = Path(VMAF_LIB_DIR, "model", f"vmaf_{VMAF_MODEL_VERSION}.json")
``` 
Given the frequency of model updates we could drop the version variable if it seems to over engineered. The current one is 3 years old.

Reply:
See the new version, I think the variable may be overkill, but otherwise your suggestion is cleaner, see what I've change it to...